src/utils/metadata.manager.ts

```diff
@@ -189,11 +189,12 @@ export class MetadataManager {
       toInsert = staticOptions ? this.addBlankLines(symbol) : `${symbol}`;
     } else {
       const text = (node as Node).getFullText(source);
-      if (text.match(/^\r?\n/)) {
-        toInsert = `,${text.match(/^\r?\n(\r?)\s+/)[0]}${symbol}`;
-      } else {
-        toInsert = `, ${symbol}`;
-      }
+      const itemSeparator = ( 
+        text.match(/^\r?\n(\r?)\s+/) ||
+        text.match(/^\r?\n/) ||
+        ' '
+      )[0];
```

Comment:
now we won't get `TypeError: Cannot read properties of null (reading '0')` anymore because if any of those `.match` returns `null`, the default value (`' '`) will be used - as `' '[0] === ' '`